### PR TITLE
Update OpenJ9 CI links in various docs

### PR DESCRIPTION
**Pull request by 0xdaryl**
Signed-off-by: Daryl Maier <maier@ca.ibm.com>

### CODE REVIEW

**Comment by 0xdaryl** [comment]
@dsouzai @pshipton : would one of you mind merging please?

File: buildenv/jenkins/README.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ Pipelines for all platforms and versions are available [**here**](https://ci.ecl
 
 #### Build
 
-Build pipelines for all platforms and versions are available [**here**](https://ci.eclipse.org/openj9/view/Build/).
+Build pipelines for all platforms and versions are available [**here**](https://openj9-jenkins.osuosl.org/view/Build/).
```
Comment (dsouzai) [inline buildenv/jenkins/README.md:220]: This link doesn't work.

File: buildenv/jenkins/README.md [hunk 2]
```diff
@@ -227,11 +227,11 @@ Build pipelines for all platforms and versions are available [**here**](https://
     - Trigger:
         - This job is used in other pipelines but can be launched manually
 
-**Note:** Windows 32 is available only for JDK8. Thus the only available build for win_x86 platform (Windows 32bits) is [Build-JDK8-win_x86](https://ci.eclipse.org/openj9/view/Build/job/Build-JDK8-win_x86/).
+**Note:** Windows 32 is available only for JDK8. Thus the only available build for win_x86 platform (Windows 32bits) is [Build-JDK8-win_x86](https://openj9-jenkins.osuosl.org/view/Build/job/Build-JDK8-win_x86/).
```
Comment (dsouzai) [inline buildenv/jenkins/README.md:230]: This link doesn't work.

File: buildenv/jenkins/README.md [hunk 3]
```diff
@@ -245,25 +245,25 @@ The [Running AdoptOpenJDK Tests](https://github.com/AdoptOpenJDK/openjdk-tests/b
...
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Promote_OMR)](https://ci.eclipse.org/openj9/job/Promote_OMR)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Promote_OMR)](https://openj9-jenkins.osuosl.org/job/Promote_OMR)
     - Description:
         - Promotes eclipse-openj9/openj9-omr branch master to branch openj9
     - Trigger:
         - Last step of `Pipeline-OMR-Acceptance`
 
 - Mirror-OpenJ9-Website-to-Eclipse
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Mirror-OpenJ9-Website-to-Eclipse)](https://ci.eclipse.org/openj9/job/Mirror-OpenJ9-Website-to-Eclipse)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Mirror-OpenJ9-Website-to-Eclipse)](https://openj9-jenkins.osuosl.org/job/Mirror-OpenJ9-Website-to-Eclipse)
```
Comment (dsouzai) [inline buildenv/jenkins/README.md:266]: This link doesn't work

**Comment by pshipton** [comment]
@AdamBrousseau should review this

**Comment by 0xdaryl** [comment]
Thanks for the review @dsouzai .  If the links have changed since moving to the new CI then I'll let @AdamBrousseau comment on what the new URLs should be.

File: buildenv/jenkins/README.md (hunk 1 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/README.md:220]:
```suggestion
Build pipelines for all platforms and versions are available [**here**](https://openj9-jenkins.osuosl.org/view/Pipelines/).
```

File: buildenv/jenkins/README.md (hunk 2 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/README.md:230]:
```suggestion
**Note:** Windows 32 is available only for JDK8. Thus the only available build for win_x86 platform (Windows 32bits) is [Build_JDK8_x86-32_windows_Nightly](https://openj9-jenkins.osuosl.org/job/Build_JDK8_x86-32_windows_Nightly/).
```

File: buildenv/jenkins/README.md (hunk 3 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/README.md:266]:
```suggestion
    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Deploy-OpenJ9-Website-to-Eclipse)](https://openj9-jenkins.osuosl.org/job/Deploy-OpenJ9-Website-to-Eclipsee)
```

File: buildenv/jenkins/README.md [hunk 4]
```diff
@@ -245,25 +245,25 @@ The [Running AdoptOpenJDK Tests](https://github.com/AdoptOpenJDK/openjdk-tests/b
...
 
 - Promote_OMR
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Promote_OMR)](https://ci.eclipse.org/openj9/job/Promote_OMR)
+    - [![Build Status](https://openj9-jenkins.osuosl.org/buildStatus/icon?job=Promote_OMR)](https://openj9-jenkins.osuosl.org/job/Promote_OMR)
     - Description:
         - Promotes eclipse-openj9/openj9-omr branch master to branch openj9
     - Trigger:
         - Last step of `Pipeline-OMR-Acceptance`
 
 - Mirror-OpenJ9-Website-to-Eclipse
```
Comment (AdamBrousseau) [inline buildenv/jenkins/README.md:265]:
```suggestion
- Deploy-OpenJ9-Website-to-Eclipse
```

File: test/docs/OpenJ9TestUserGuide.md [hunk 5]
```diff
@@ -606,16 +606,16 @@ A common scenario is that automated testing finds a failure and a developer is a
...
 * SDK build that was used by the test (in the console output of the Jenkins job, there is java -version info and a link to the SDK used)
 
-A specific example, [Issue 6555](https://github.com/eclipse-openj9/openj9/issues/6555) Test_openjdk13_j9_sanity.system_ppc64le_linux TestIBMJlmRemoteMemoryAuth_0 crash 
+A specific example, [Issue 6555](https://github.com/eclipse-openj9/openj9/issues/6555) Test_openjdk13_j9_sanity.system_ppc64le_linux TestIBMJlmRemoteMemoryAuth_0 crash
 we get the following info (captured in the name of the issue):
 * TARGET = TestIBMJlmRemoteMemoryAuth_0
 * BUILD_LIST = system
 * JDK_VERSION = 13
-* JDK_IMPL = openj9 (its implied if the failure was found in openj9 testing at ci.eclipse.org/openj9)
+* JDK_IMPL = openj9 (its implied if the failure was found in openj9 testing at openj9-jenkins.osuosl.org)
```
Comment (AdamBrousseau) [inline test/docs/OpenJ9TestUserGuide.md:614]: ?
```suggestion
* JDK_IMPL = openj9 (its implied if the failure was found in openj9 testing at OpenJ9's Jenkins)
```
?

**Review by AdamBrousseau — APPROVED** [review]
LGTM pending broken links being fixed.

File: buildenv/jenkins/README.md (hunk 3 above)
Comment (0xdaryl) [reply buildenv/jenkins/README.md]: Will fix in a rebase as there's a typo in the URL.

**Comment by 0xdaryl** [comment]
@AdamBrousseau's changes applied and squashed.  Ready for another review please.

**Review by dsouzai — APPROVED** [review]

**Pull request merged**